src/components/Header/NavBar/NavBar.js

```diff
@@ -42,8 +42,8 @@ const mapStateToProps = state => ({
   auth: state.auth,
 });
 
-export const ConnectedNavBar = connect(mapStateToProps,
-  { handleLogOut: logOut })(NavBar);
+export const ConnectedNavBar = withRouter(connect(mapStateToProps,
```

Comment:
Why we need `withRouter` hoc here?

Reply:
I need withRouter, because NavLink matches url to add the active class.